RaceTrack/Tournament/Tournament.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DataRace;
+
+namespace RaceTournament
+{
+    public class TournamentResult
+    {
+        private Queue<Racer> _racers;
+        public Queue<Racer> _winners = new Queue<Racer>();
```

Comment:
May be for your implementation you will need `ConcurrentQueue` and/or `SemaphoreSlim`